conf/ds-present-environment.yaml.SAMPLE

```diff
@@ -0,0 +1,125 @@
+#
+# This contains behaviour data: Thread allocation, allowed fields for lookup, limits for arguments etc.
+#
+# The behaviour config is normally controlled by developers and is part of the code repository.
+# Sensitive information such as machine names and user/passwords should not be part of this config.
+#
+# It will be automatically merged with the environment config when accessed through the
+# application config system.
+#
+#
+
+
+# Image server URLs are injected into transformations.
+# These should be overridden in environment specific config files for all environments, meaning local test,
+# shared devel, stage and production
+#
+# Note: The server marked with "default: true" is used in the base configuration ds-present-kb-collections.yaml
+ # Backing image servers for iip, iiif and deepzoom protocols
+ # At KB the same IIPImage server(s) is used for all three protocols
+ imageservers:
+   iip:
+     server: 'https://<iipimageserver>.kb.dk/'
+   iiif:
+     server: 'https://<iipimageserver>.kb.dk/'
```

Comment:
This should be marked as `default: true` and to avoid problems I think the `default: true` in the behaviour should be commented out, with a comment stating that it is commented out to force defining a default for a real imageserver in the local, devel or prod config